### ISSUE TITLE
Dropbox dependency needs to be updated to NPM dropbox 2.0.0 or higher

### ISSUE THREAD

**Issue by michellarcari** [CollectionFS/Meteor-CollectionFS#1015]
As of today, Dropbox API v1 is no longer supported, thus, the cfs dropbox package needs to migrate the NPM dependency from 0.10.3 to 2.0.0 or higher (Dropbox API v2).
I tried to change the dependency only but it breaks the code. If I can make it work out fine I will send a pull request, but I have limited time to work on it.

**Comment by Sojourneer** [comment]
The API is very different, so it will take a good understanding to change the cfs:dropbox shim.  I might just wind up moving off cfs because of its many issues and the project basically being dead.

**Comment by harryadel** [comment]
@Sojourneer You're absolutely right, we need to migrate to the latest NPM version and modify code accordingly. But as of right now, we're aiming to bring the packages(as best as we can) to a stable state before making any big updates. So, please if you can help us out, don't hesitate to create a PR.
